store/remote_test.go

```diff
@@ -46,19 +48,26 @@ func TestNewRemote(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unexpected error reading index: %v", err)
 	}
+
 	if !ok {
-		t.Fatalf("unexpected index not found")
+		t.Errorf("unexpected index not found")
```

Comment:
Are those s/Fatalf/Errorf/ there to get a deeper/meaningful view on failures? (Not objecting them, I'm just unsure about the rationale)